helpers/test.js

```diff
@@ -14,4 +14,12 @@ module.exports = {
     city: 'New Orleans',
     state: 'LA',
   },
+  stringifyUser: function neo4jStringify1(object) {
```

Comment:
This is awesome! We can simplify this even more by adding a type to the parameters: `neo4jStringify(object, type)...` then use the type for the neo4j variable name: `\`${type}.${key}= ...`